routing/routing.xml

```diff
@@ -740,74 +1074,74 @@
 
 		<way attribute="obstacle_srtm_alt_speed">
 			<if param="height_obstacles">
-				<gt value1="-100" value2=":incline">
```

Comment:
Is it correct change from semicolon to dollar-sign? It has different semantics

Reply:
Bit of history: I did a PR earlier where I fucked up some binary files, so I created a cleaner version.

There was a lot of discussion about all these details in #460. One of them is [this comment](https://github.com/osmandapp/OsmAnd-resources/pull/460#issuecomment-363588384), which states that they are the same.

I could be mistaken of course, in that case, I'll gladly here what the difference in semantics are.

Reply:
I don't understand where did you get there is no difference:
- $maxweight - is the value of the tag "maxweight" of 
- : incline is a routing parameter and it is specified in routing settings
-<gt value1=":height" value2="$maxheight" type="length">

Reply:
No, there is no difference. Please see [this comment](https://github.com/osmandapp/OsmAnd-resources/pull/460#issuecomment-363588384)